udata_front/theme/gouvfr/assets/js/locales/en.json

```diff
@@ -449,7 +449,7 @@
   "Please only report in case of serious concern.": "Please only report in case of serious concern.",
   "See our usage policy": "See our usage policy",
   "Report reason": "Report reason",
-  "Reason of your report.\\nDon\\'t include any personal data.": "Reason of your report.\\nDon\\'t include any personal data.",
+  "Reason of your report.{newline}Don't include any personal data.": "Reason of your report.\nDon't include any personal data.",
```

Comment:
I'm not sure why, but when running `inv i18n` again, it removes the value? Should we recompile these with Crowdin?

Reply:
I think this was the case because I didn't update all the files. It's working now that all files have the same keys.